Gemfile

```diff
@@ -1,7 +1,8 @@
 source 'https://rubygems.org'
 
 
-
+# validation email
+gem 'validates_formatting_of'
```

Comment:
Please remove this gem from Gemfile if you don't use it :construction_worker_man: